manim/animation/creation.py

```diff
@@ -200,6 +203,87 @@ def __init__(
         super().__init__(mobject, rate_func=rate_func, remover=remover, **kwargs)
 
 
+class Add(Animation):
+    """Adds a Mobject to the scene with no animated effect. This is functionally equivalent to :meth:`~.Scene.add`
+
+    Parameters
+    ----------
+    mobject : :class:`~.Mobject`
+        The Mobject to add.
+
+    Raises
+    ------
+    :class:`TypeError`
+        If ``mobject`` is not an instance of :class:`~.Mobject` or :class:`~.OpenGLMobject`
+
+    Examples
+    --------
+    .. manim:: AddScene
+
+        class AddScene(Scene):
+            def construct(self):
+                self.wait(0.5)
+                self.play(Add(Square()))
+
+    See Also
+    --------
+    :class:`~.Create`
+    :meth:`~.Scene.add`
+
+    """
+
+
+class Remove(Animation):
+    """Removes a Mobject from the scene with no animated effect. This is functionally equivalent to :meth:`~.Scene.remove`
+
+    Parameters
+    ----------
+    mobject : :class:`~.Mobject`
+        The Mobject to remove.
+
+    Raises
+    ------
+    :class:`TypeError`
+        If ``mobject`` is not an instance of :class:`~.Mobject` or :class:`~.OpenGLMobject`
+
+    Examples
+    --------
+    .. manim:: RemoveScene
+
+        class RemoveScene(Scene):
+            def construct(self):
+                square = Square()
+                self.add(square)
+                self.wait(0.5)
+                self.play(Remove(square))
+                self.wait(0.5)
+
+    See Also
+    --------
+    :class:`~.Uncreate`
+    :meth:`~.Scene.remove`
+
+    """
+
+    def __init__(self, mobject: Union[Mobject, OpenGLMobject]) -> None:
+        super().__init__(mobject)
```

Comment:
This class could just have a contructor that would put remover=True to the super constructor, no ?

Reply:
Had this same thought when I updated `Add` to inherit and do nothing and tried this. Appears that `remover=True` is broken cause it did nothing.

Reply:
This is .. weird. 

At least you can remove the other methods that are just "blankly" overriden ;)

Reply:
I removed those methods and opened #1582 in regards to the other problem.